Runtime upgrades section (#1525)
w3f/polkadot-wiki: docs/build-protocol-info.md

transaction data using the
[SCALE codec](https://substrate.dev/docs/en/knowledgebase/advanced/codec).

## Runtime Upgrades

Runtime upgrades allow Polkadot to change the logic of the chain without the need for a hard fork. A

> hard fork would require node operators to manually upgrade their nodes to the latest Runtime

Reviewer: ```suggestion
hard fork would require node operators to manually upgrade their nodes to the latest runtime
```

> version. In a distributed system, this is a complex process to coordinate and communicate.  
> 
> Polkadot upgrades are generally forkless. The existing runtime logic is followed to update the Wasm

Reviewer: ```suggestion
version. In a distributed system, this is a complex process to coordinate and communicate. Polkadot can upgrade without a hard fork. The existing runtime logic is followed to update the Wasm
```

Reviewer: Pull these two paragraphs into one (will need to re-run prettier)
